test/contractBridge/whitelist.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright 2020 ChainSafe Systems
+ * SPDX-License-Identifier: LGPL-3.0-only
+ */
+
+const TruffleAssert = require('truffle-assertions');
+
+const BridgeContract = artifacts.require("Bridge");
+const TestWhitelistContract = artifacts.require("TestWhitelist");
+
+contract('Bridge - [whitelist]', async accounts => {
+    const chainID = 1;
+    const initialRelayers = accounts.slice(0, 3);
+    const initialRelayerThreshold = 2;
+
+    const adminAccount = accounts[0];
+    const genericAccount = accounts[3];
+    const testAccount = accounts[4];
+
+    let BridgeInstance;
+    let ADMIN_ROLE;
+    let WHITELISTER_ROLE;
+
+    beforeEach(async () => {
+        BridgeInstance = await BridgeContract.new(chainID, initialRelayers, initialRelayerThreshold, 0, 100);
+        ADMIN_ROLE = BridgeInstance.DEFAULT_ADMIN_ROLE;
+        WHITELISTER_ROLE = BridgeInstance.WHITELISTER_ROLE;
+    });
+
+    // Testing whitelist methods
+
+    it('should be disabled by default', async () => {
```

Comment:
Shouldn’t it be enabled by default? This way we don’t need to send a second transaction to enable it and we are sure that no deposit can ever come through before we allow it.

Reply:
Yes, it should. I [enabled](https://github.com/Cerebellum-Network/chainbridge-solidity/pull/6/files#diff-d1d675d04151a0a3ec00bad5e7e2fbea4c4d2e70a75a00f656df8fa08553f5f7R52) it and [updated](https://github.com/Cerebellum-Network/chainbridge-solidity/pull/6/files#diff-a62c2a6bda93583bed6bafc173326ced2d611a6ae8a37740d97001a613dceefeR32-R34) the tests.